### PR TITLE
Vietnamese translation

### PR DESCRIPTION
**Pull request by taozherotoxic**
from Vietnam with love <3

### CODE REVIEW

File: src/_locales/vi.config [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+#==== Top section ====
+
+@on
+Bật
+
+@off
+Tắt
+
+@toggle_current_site
+Chuyển đổi trang web này
```
Comment (Gusted) [inline src/_locales/vi.config]: Remove the whitesapce after `này` So it pass the checks :)

Comment (Gusted) [reply src/_locales/vi.config]: Highlighted text I mean with the whitespace

Comment (alexanderby) [reply src/_locales/vi.config]: You can continue editing your Pull Request. But if there's no other mistakes I will fix it myself.

**Comment by alexanderby** [comment]
Will be released soon.

**Pull request merged**